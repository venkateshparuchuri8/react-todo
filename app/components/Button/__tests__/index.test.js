/**
 * @author vishwadeep.kapoor
 * @Date: 2018-12-26 12:18:44
 * @Last Modified by: vishwadeep
 * @Last Modified time: 2019-01-14 16:30:43
 */

/* eslint-disable no-undef */
import React from 'react';
import Button from '../index';

describe('<Button /> rendering', () => {
  it('is renders correctly', () => {
    const tree = renderWithTheme(<Button />);
    expect(tree).toMatchSnapshot();
  });
});
