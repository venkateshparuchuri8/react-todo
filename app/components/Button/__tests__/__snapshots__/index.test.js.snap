// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> rendering is renders correctly 1`] = `
<button
  className="sc-bdVaJa iICpcp"
/>
`;
